docs/conf.py

```diff
@@ -28,6 +28,16 @@
 
 import sphinx.environment
 
+_warn_node_old = sphinx.environment.BuildEnvironment.warn_node
```

Comment:
you don't need it anymore. check latest cookiecutter-invenio-module (upgrade Sphinx).